server/builds.go

```diff
@@ -33,12 +32,12 @@ func (b *Builds) getInstallationVersion(pr *model.PullRequest) string {
 
 func (b *Builds) dockerRegistryClient(s *Server) (reg *registry.Registry, err error) {
 	if _, err = url.ParseRequestURI(s.Config.DockerRegistryURL); err != nil {
-		return nil, errors.Wrap(err, "invalid url for docker registry")
+		return nil, fmt.Errorf("invalid url for docker registry")
```

Comment:
There is a general consensus on using `github.com/pkg/errors` for error handling and wrapping. It's not documented jet, [but on it's way](https://github.com/mattermost/mattermost-developer-documentation/issues/612). Given that this codebase uses `github.com/pkg/errors` already a lot, I'm wondering why there is the need to move away from it.

Reply:
Yeah, maybe I am too optimistic that go2 will be released soon. I will revert this.